scss/_variables.scss

```diff
@@ -61,12 +61,17 @@ $indigo: #6610f2 !default;
 $purple: #6f42c1 !default;
 $pink: #e83e8c !default;
 $red: #ff140f !default;
-$orange: #fd7e14 !default;
+// $orange: #fd7e14 !default;
```

Comment:
can we delete these instead of commenting them out?

Reply:
-No problem